pkg/core/dao/dao.go

```diff
@@ -180,21 +180,75 @@ func (dao *Simple) putTokenTransferInfo(acc util.Uint160, bs *state.TokenTransfe
 
 // -- start transfer log.
 
-func getTokenTransferLogKey(acc util.Uint160, index uint32, isNEP11 bool) []byte {
-	key := make([]byte, 1+util.Uint160Size+4)
+func getTokenTransferLogPrefix(acc util.Uint160, isNEP11 bool) []byte {
```

Comment:
Not really needed. You can save some allocations by always using `getTokenTransferLogKey`.